pkg/mapper/configmap/configmap.go

```diff
@@ -25,9 +26,11 @@ import (
 )
 
 type MapStore struct {
-	mutex sync.RWMutex
-	users map[string]config.UserMapping
-	roles map[string]config.RoleMapping
+	mutex        sync.RWMutex
+	users        map[string]config.UserMapping
+	userArnLikes map[string]config.UserMapping
+	roles        map[string]config.RoleMapping
+	roleArnLikes map[string]config.RoleMapping
```

Comment:
I'm not following why you'd need to have separate `userArnLikes` and `roleArnLikes` fields here. Now that you've placed the `UserArnLike` and `RoleArnLike` fields within the `UserMapping` and `RoleMapping` structs respectively I'm unsure why you would need a separate collection of those structs here in `MapStore`?